src/utils/constants.js

```diff
@@ -37,3 +37,10 @@ export const MODEL_SERVER_PROPS =  [
 ];
 
 export const MODEL_SERVER_PROPS_ALL = MODEL_SERVER_PROPS.concat(['role','id']);
+
+export const PLAYBOOK_PROGRESS_UI_STATUS = {
```

Comment:
Hmmm. I would rather not introduce yet another set of STATUS variables with slightly different names (``NOTSTARTED`` instead of ``NOT_STARTED``) and values (strings instead of integers). Why can't we just use ``STATUS`` values instead.

Reply:
This is for UI classes to turn red, green gray for the item listed...just kind of reuse the existing. Will look into it.

Reply:
Yeah...I will see what I can do.